### PR TITLE
Oryx - Clean up and shrink size

### PR DESCRIPTION
**Pull request by samruddhikhandale**
With this cleanup, the size of the docker layer for the oryx feature reduced from `2.6 GB` to `205 MB`

Before these changes

image

After

image

### CODE REVIEW

File: src/oryx/devcontainer-feature.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "id": "oryx",
-    "version": "1.0.1",
+    "version": "1.0.2",
```
Comment (joshspicer) [inline src/oryx/devcontainer-feature.json:3]: This is going to have to get bumped one higher with the change I just merged

**Comment by joshspicer** [comment]
Re: the oryx/jammy test missing libssl:

I spent (too much) time debugging this in the `dotnet` feature a month or so ago.  I left some comments in the script that might be useful

https://github.com/devcontainers/features/blob/main/src/dotnet/install.sh#L328

**Comment by samruddhikhandale** [comment]
> Re: the oryx/jammy test missing libssl:
> 

Even with installed `libssl` packages, the `oryx build` and `oryx prep` cmd fails. It looks like a upstream oryx issue. I have asked them about it in chat if they are aware, else I'll create issues on `oryx` and this repo for tracking. 

We are realizing this 🐛  now because I added `oryx prep` in the `test.sh` in this PR. 

image

**Comment by samruddhikhandale** [comment]
>  It looks like a upstream oryx issue. I have asked them about it in chat if they are aware.

Spoke with the team, oryx is only currently supported `ubuntu focal scm` based build image

**Review by joshspicer — APPROVED** [review]
Chatted offline.


After adding a warning in the oryx/install.sh noting no support for jammy, this is good to override merge with the failing oryx jammy test.

We can leave it as another action item for https://github.com/devcontainers/cli/issues/139 :)

**Pull request merged**